items/spell_hopper/index.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "spell_hopper",
+  "displayName": "Hopper Spell",
+  "type": "",
```

Comment:
Correct me if I'm wrong, but `type` is set to an empty string because we don't actually plan on having the player equip them, right? I saw that all of the spells are set up this way, so I thought I'd double check instead of assuming anything haha